release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -705,6 +712,16 @@ This script removes unauthenticated subjects from the following cluster role bin
 // TODO: This known issue should carry forward to 4.9 and beyond!
 * The `oc annotate` command does not work for LDAP group names that contain an equal sign (`=`), because the command uses the equal sign as a delimiter between the annotation name and value. As a workaround, use `oc patch` or `oc edit` to add the annotation. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1917280[*BZ#1917280*])
 
+* Cluster administrators can specify a custom error code response page for either 503, 404, or both error pages. The router does not reload to reflect custom error code pages updates. As a workaround, rsh in the router pods and run `reload-haproxy` in all the router pods that serve the custom http error code pages:
```

Comment:
Cluster administrators can specify a custom **http** error code response page for either 503, 404, or both error pages